app/api/server/v1/teams.ts

```diff
@@ -62,15 +65,75 @@ API.v1.addRoute('teams.create', { authRequired: true }, {
 
 API.v1.addRoute('teams.members', { authRequired: true }, {
 	get() {
-		const { teamId } = this.queryParams;
+		if (!hasAtLeastOnePermission(this.userId, ['add-team-member', 'edit-team-member', 'view-all-teams'])) {
+			return API.v1.unauthorized();
+		}
```

Comment:
```suggestion

```
Team members should be visible to everyone on the team regardless of permissions.
I'll include the proper validation on a separate pull request.